.travis.yml

```diff
@@ -5,16 +5,51 @@ php:
   - 5.4
   - 5.5
   - 5.6
-  - 7.0
   - hhvm
-  
+  - nightly
+
+env:
+  - SYMFONY_VERSION=2.6.*
+
 matrix:
+  fast_finish: true
+  include:
+    - php: 5.3
+      env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest"
```

Comment:
``--prefer-lowest`` should be used without changing the symfony requirements, otherwise we don't ensure we have the right requirements